docs/detections/rules-ui-create.asciidoc

```diff
@@ -311,7 +313,9 @@ NOTE: Rules that use aggregating queries might create duplicate alerts. This can
 [float]
 [[esql-non-agg-query]]
 ===== Non-aggregating query 
-Non-aggregating queries doesn't use `STATS...BY` functions and doesn't aggregate source event data. Alerts generated by an {esql} rule with a non-aggregating query only contain the fields returned by the query.
+Non-aggregating queries don't use `STATS...BY` functions and don't aggregate source event data. Alerts generated by a non-aggregating query contain source event fields that the query returns, new fields the query creates, and all other fields in the source event document. 
```

Comment:
I would go with the first revision, since it says firstly fields from query + rest source fields.

And that is more correct, since if any field has duplicate name, the returned from query would have a priority. And only missing (rest of the source doc fields) will be merged into alert